flytekit/clis/sdk_in_container/run.py

```diff
@@ -293,3 +367,11 @@ def encode(
             uri=df_remote_location.native_url[: -len(filename)],
             metadata=literals.StructuredDatasetMetadata(structured_dataset_type),
         )
+
+
+run = RunCommand(
+    name="run",
+    help="Run_old command, a.k.a. script mode. It allows for a a single script to be "
+    + "registered and run from the command line or any interactive environment "
```

Comment:
```suggestion
    + "registered and run from the command line. "
```

This does not help to run it from jupyter, lets drop that

Reply:
done